server/app.yaml

```diff
@@ -0,0 +1,15 @@
+# Google App Engine config file
```

Comment:
Do we really need all these comments here?

Reply:
Removed relevant config files for google cloud.